rest/route/middleware.go

```diff
@@ -436,15 +445,21 @@ func urlVarsToProjectScopes(r *http.Request) ([]string, int, error) {
 	taskID := util.CoalesceStrings(append(query["task_id"], query["taskId"]...), vars["task_id"], vars["taskId"])
 	if projectID == "" && taskID != "" {
 		projectID, err = task.FindProjectForTask(taskID)
+		grip.Infof("middleware.go, 439. ProjectID: %s", projectID)
 		if err != nil {
 			return nil, http.StatusNotFound, err
 		}
 	}
 
+	projectRef, err := model.FindOneProjectRef(projectID)
+	if err == ErrProjectNotFound {
```

Comment:
It's quite confusing (and maybe not working) that this variable is declared (publicly) in two packages. I suspect this won't actually work the way that you want it to.

Reply:
Thanks! fixing.